utils/mongo-utility.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+var mongoose = require('mongoose');
+
+var demoSchema;
+var DemoModel;
+
+var db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', function() {
+  console.log('connection established');
+
+  // create schema
+  demoSchema = mongoose.Schema({
+    Category: {
+      type: String,
+      required: true
+    },
+    CategoryID: {
+      type: String,
+      required: true
+    },
+    MetaTitle: String,
+    MetaDescription: String
+  });
+  // create model
+  DemoModel = mongoose.model('Demo', demoSchema);
+},{
+  timestamp: true
+});
+
+mongoose.connect('mongodb://localhost/test');
```

Comment:
Would be better if you can create the mongo url based on a config variable. So that we would be able to switch mongo servers based on the environment.